_sass/base/_override.scss

```diff
@@ -0,0 +1,96 @@
+
+// Override Bootstrap defaults
+body {
+  color: $textColor;
+  font-family: "Roboto", "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+a {
+  color: $mainBrandColor;
+  font-weight: 400;
+}
+
+a:hover,
+a:focus {
+  color: $linkHoverColor;
+}
+
+a.external {
+  &:after {
+    @extend .external-link-icon;
+    content: "";
+    margin-left: 5px;
+    display: inline-block;
+  }
+
+  &:hover {
+    &:after {
+      background: url('../img/external-link-dark.svg') no-repeat center/cover;
+    }
+  }
+}
+
+a.disabled {
+    color: #CCCCCC;
+}
+
+p,
+li {line-height: 22px;}
+
+p.note {
+  background-color: $thirdBrandColor;
+  padding: 9.5px;
+}
+
+p.note code {
+  background-color: $thirdBrandColor !important;
+}
+
+code {
+  color: $codeColor;
+  background-color: $inverseBrandColor;
+  font-size: 14px;
+}
+
+pre, pre.prettyprint {
```

Comment:
Just based on your local Wildebeest preview, pre elements (and, I think, code elements) are rendering in normal text font rather than code font/Courier (it took me a while to figure out why they looked odd....). I know I didn't have to specify it in my original CSS but it seems to be no longer the default...